logging/cluster-logging-upgrading.adoc

```diff
@@ -5,18 +5,27 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+To upgrade to OpenShift Logging 5.0, you update the {product-title} cluster from 4.6 to 4.7. Then, you update the following operators:
 
+* From Elasticsearch Operator 4.6 to OpenShift Elasticsearch Operator 5.0
```

Comment:
There is not an explicit need to upgrade to 5.0 before others.  User's can update to any 5.x available log choice

Reply:
Updated. Thanks.